src/java/org/apache/cassandra/db/memtable/pmem/PmemPartition.java

```diff
@@ -375,7 +391,8 @@ public UnfilteredRowIterator unfilteredIterator(ColumnFilter columns, Slices sli
     @Override
     public UnfilteredRowIterator unfilteredIterator(ColumnFilter columns, NavigableSet<Clustering<?>> clusteringsInQueryOrder, boolean reversed)
     {
-        if (clusteringsInQueryOrder.isEmpty() || block.getLong(DELETION_INFO_OFFSET) == -1)
+       // if (clusteringsInQueryOrder.isEmpty() || block.getLong(DELETION_INFO_OFFSET) == -1)
+        if (block.getLong(DELETION_INFO_OFFSET) == -1)
```

Comment:
These bytes are used to mark tombstone if partition has been deleted. 
In the latest update, you will find 
- 0, if partition is live 
- handle to the MemoryBlock containing the deletion info, otherwise